src/actions/index.js

```diff
@@ -159,3 +160,18 @@ export const fetchItems = category => {
       });
   };
 };
+
+export const loadSingleItem = item => {
+  return dispatch => {
+    return fetch('/api/items/:id')
```

Comment:
This is where you would want to replace `:id` with an actual id based on the detail being viewed.